netci.groovy

```diff
@@ -116,13 +117,29 @@ set EchoOn=true
 BuildAndTest.cmd -build:true -clean:false -deployExtensions:true -trackFileAccess:false -officialBuild:false -realSignBuild:false -parallel:true -release:true -delaySignBuild:true -samples:false -unit:false -eta:false -vs:true -cibuild:true -x64:false -netcoretestrun
 popd""")
 
-       // Revert patched targets and rules from backup.
-        batchFile("""SET VS_MSBUILD_MANAGED=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\Managed
+           // Revert patched targets and rules from backup.
+            batchFile("""SET VS_MSBUILD_MANAGED=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\Managed
 del /SQ "%VS_MSBUILD_MANAGED%\\"
 xcopy /SIY .\\backup\\Managed "%VS_MSBUILD_MANAGED%"
 rmdir /S /Q backup
 """)
+        }
+    }
+
+    addVsiArchive(newVsiJob)
+    Utilities.setMachineAffinity(newVsiJob, 'Windows_NT', 'latest-or-auto-dev15-internal')
+    Utilities.standardJobSetup(newVsiJob, project, isPR, "*/${branch}")
+    // ISSUE: Temporary until a full builder for source control is available.
```

Comment:
@mmitche Are you sure that PR jobs will not work with current `addVsiMultiScm` implementation? Otherwise, I can try merging this PR and test if it works or not.